__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should expect TARGET_MIGRATION_VERSION to be 7', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+  it('TEST should expect TARGET_MIGRATION_VERSION to be 8', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case has been updated to reflect the new `TARGET_MIGRATION_VERSION` value. This change is expected if a new database migration has been introduced, which seems to be the case given the context provided in the PR summary.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```
Please ensure that the new migration version is correctly handled in your migration scripts and that all necessary changes to the database schema have been made.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Yes it's correct, the last migration file is 8